Packs/GmailSingleUser/Integrations/GmailSingleUser/README.md

```diff
@@ -1,63 +1,75 @@
+Below are the steps to create OAuth 2.0 authorization flow.\
+To use the Demisto app, follow the steps mentioned in the 'GSuite Admins' section.\
+To create your own app, follow the steps mentioned in the 'All Account Types' section.\
+Once you have the app, please follow the 'Authorization Flow In Cortex XSOAR' section.
 
-1. Create and save an integration instance of the Gmail Single User integration. Do not fill in the *Auth Code* field, this will be obtained in the next steps.
-2. To obtain the **Auth Code** run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account. 
-3. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
-4. Save the instance.
-5. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
+### GSuite Admins:
 
 **NOTE:** The Demisto App is going through the Google verification process. During the verification process the app is not fully verified, and you may receive from Google an "unverified app" warning in the authorization flow.
 
-**GSuite Admins:** You can choose to trust the Demisto App so your users can configure the App. Instructions:
+You can choose to trust the Demisto App so your users can configure the App. Instructions:
 * Go to [App Access Control](https://admin.google.com/ac/owl/list?tab=apps)
 * Choose: `Configure new app` -> `OAuth App Name Or Client ID`. 
   ![GSuite App Configurations](doc_imgs/gsuite-configure-app.png)
 * Enter the following Client ID: `391797357217-pa6jda1554dbmlt3hbji2bivphl0j616.apps.googleusercontent.com`
 * You will see the `Demisto App` in the results page.
   ![Demisto App](doc_imgs/demisto-app-result.png)
-* Select the App and grant the App access as `Trusted`. 
+* Select the App and grant the App access as `Trusted`.
+* Add the Demisto app client ID `391797357217-pa6jda1554dbmlt3hbji2bivphl0j616.apps.googleusercontent.com` to the integration configuration 
+* Proceed to the 'Authorization Flow In Cortex XSOAR' section to configure the OAuth 2.0 authorization in Cortex XSOAR.
 
 Additional info available at: https://support.google.com/a/answer/7281227
 
-**All Acount Types:**
+### All Account Types:
 
-* **Optional**: You can use your own Google App instead of the default Demisto App. To create your own app, follow the [Google instructions for Desktop Apps](https://developers.google.com/identity/protocols/OAuth2InstalledApp#prerequisites). 
-
-* Go to the developers credentials page: https://console.developers.google.com/apis/credentials (you may need to setup a new project if you haven't done so in the past).
-* If needed, configure the Consent Screen. Fill in the Consent Screen information you would like to display to your users.
+* **Optional**: You can use your own Google App instead of the default Demisto App. Follow the steps below.
```

Comment:
this is not optional anymore.
If they are not GSuiteAdmin, their only option is to create their own app

Reply:
fixed